clang/lib/StaticAnalyzer/Checkers/CStringChecker.cpp

```diff
@@ -273,36 +273,41 @@ class CStringChecker : public Checker< eval::Call,
   /// Invalidate the destination buffer determined by characters copied.
   static ProgramStateRef
   invalidateDestinationBufferBySize(CheckerContext &C, ProgramStateRef S,
-                                    const Expr *BufE, SVal BufV, SVal SizeV,
-                                    QualType SizeTy);
+                                    const Expr *BufE,
+                                    CFGBlock::ConstCFGElementRef ElemRef,
```

Comment:
In all of these examples, I feel that `ElemRef` is too verbose. `Elem` or `Place` should be a shorter name without compromising readability. Also, like I previously suggested, not having to spell out the `CFGBlock::` would be a huge help here too.